### DISCUSSION TITLE
Using Nanoid in "type": "module"

### DISCUSSION THREAD

**Discussion by TinyTyno** [ai/nanoid#475] [Q&A] [answered]
I faced this error :
> The requested module 'nanoid' does not provide an export named 'default'
I checkd the nanoid.js file and realise that the code indeed does not provide "default", how do i solve this issue?

**Comment by ai** [comment, marked as answer]
Use `import { nanoid } from 'nanoid'` (named import) instead of `import nanoid from 'nanoid'` (default import).
